argon2/src/error.rs

```diff
@@ -116,6 +120,7 @@ impl From<Error> for password_hash::Error {
             Error::ThreadsTooMany => InvalidValue::TooLong.param_error(),
             Error::TimeTooSmall => InvalidValue::TooShort.param_error(),
             Error::VersionInvalid => password_hash::Error::Version,
+            Error::AllocFail => InvalidValue::TooLong.param_error(),
```

Comment:
I am not sure what to do here. Maybe we should add a new error variant to `password_hash::Error`?

Reply:
Sure